iOS/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -177,6 +230,26 @@ extension CourseListViewController: UISearchControllerDelegate {
             statusBarBackground.autoresizingMask = [.flexibleWidth]
             self.statusBarBackground = statusBarBackground
         }
+
+        self.updateSearchFilterContainerHeight(isSearching: true)
+        self.collectionViewLayout.invalidateLayout()
+
+        // swiftlint:disable:next trailing_closure
```

Comment:
Why disabling linting here?

Reply:
When using a trailing closure, swift compiler can't resolve the method 😞